exporter/collector/metrics.go

```diff
@@ -657,7 +657,7 @@ func (m *metricMapper) histogramToTimeSeries(
 	point pmetric.HistogramDataPoint,
 	projectID string,
 ) []*monitoringpb.TimeSeries {
-	if point.Flags().NoRecordedValue() || !point.HasSum() {
+	if point.Flags().NoRecordedValue() || !point.HasSum() || point.ExplicitBounds().Len() == 0 {
```

Comment:
Can we add a debug-level log?  It would also be nice if it was possible to use self-obs metric to detect cases like these where we drop a metric, but we shouldn't block this bug fix for that.

Reply:
sure, added